ui/appui-react/src/appui-react/frontstage/FrontstageDef.tsx

```diff
@@ -274,7 +280,10 @@ export class FrontstageDef {
     }
 
     for (const popoutId of popoutsToOpen) {
-      this.openPopoutWidgetContainer(popoutId);
+      if (oldState) {
```

Comment:
Haven't tried reproducing, but seems like that's the case while looking at the code.

Reply:
Added a fix to push the old state requirement deeper into the code. An electron popout should still popout on original state. I just worry about the edge case with it where the popout fails on the initial state on Electron, that would lead to the widget disappearing again